Webpack 5 upgrade script error

Team, `webpack` version 5 and `webpack-cli` version 4 release yesterday Oct 10, it breaks any new projects created using `framework7-cli` since code gen tends to consume latest version of both libraries. Since webpack newer release is major release, it has API breaking changes.

```
internal/modules/cjs/loader.js:968
  throw err;
  ^

Error: Cannot find module 'webpack-cli/bin/config-yargs'
```
Until framework-cli is compatable with `webpack@5` following commands will get the users going

```
npm uninstall webpack webpack-cli
npm install webpack@4 webpack-cli@3
```

I hope this helps for others.

--- a/create/utils/generate-package-json.js
+++ b/create/utils/generate-package-json.js
@@ -68,8 +68,8 @@ module.exports = function generatePackageJson(options) {
       ] : []),
       'terser-webpack-plugin',
       'url-loader',
-      'webpack',
-      'webpack-cli',
+      'webpack@4',
+      'webpack-cli@3',
       'webpack-dev-server',
       ...(type.indexOf('pwa') >= 0 ? [
         'workbox-webpack-plugin',

